entities/global.ent

```diff
@@ -362,160 +362,160 @@
-<!ENTITY url.php.talks 'http://talks.php.net/'>
+<!ENTITY url.php.talks 'https://talks.php.net/'>
 <!ENTITY url.php.support "https://www.php.net/support.php">
 <!ENTITY url.php.urlhowto "https://www.php.net/urlhowto.php">
 <!ENTITY url.phpdbg.docs "https://phpdbg.room11.org/">
-<!ENTITY url.phpeditorlist "http://en.wikipedia.org/wiki/List_of_PHP_editors">
-<!ENTITY url.powerarchiver "http://www.powerarchiver.com/">
+<!ENTITY url.phpeditorlist "https://en.wikipedia.org/wiki/List_of_PHP_editors">
+<!ENTITY url.powerarchiver "https://www.powerarchiver.com/">
 <!ENTITY url.pslib "http://pslib.sourceforge.net/">
 <!ENTITY url.pspell "http://aspell.net/">
-<!ENTITY url.qdbm "http://fallabs.com/qdbm/index.html">
-<!ENTITY url.tcadb "http://fallabs.com/tokyocabinet/">
-<!ENTITY url.qmail "http://cr.yp.to/qmail.html">
+<!ENTITY url.qdbm "https://fallabs.com/qdbm/index.html">
+<!ENTITY url.tcadb "https://fallabs.com/tokyocabinet/">
+<!ENTITY url.qmail "https://cr.yp.to/qmail.html">
 <!ENTITY url.quickzip "http://www.quickzip.org/">
-<!ENTITY url.rar "http://www.rarlabs.com/">
+<!ENTITY url.rar "https://www.rarlabs.com/">
 <!ENTITY url.readline "http://cnswww.cns.cwru.edu/~chet/readline/rltop.html">
 <!ENTITY url.readline.windows "http://mingweditline.sourceforge.net/">
-<!ENTITY url.recode "http://directory.fsf.org/All_GNU_Packages/recode.html">
+<!ENTITY url.recode "https://directory.fsf.org/All_GNU_Packages/recode.html">
 <!ENTITY url.rec-xml "http://www.w3.org/TR/1998/REC-xml-19980210">
 <!ENTITY url.relaxng "http://www.relaxng.org/">
 <!-- linking to specific rfcs is done like so: &url.rfc;xxxx so for example &url.rfc;2042 -->
 <!ENTITY url.rfc  'http://www.faqs.org/rfcs/rfc'>
-<!ENTITY url.iana.system-names 'http://www.iana.org/assignments/operating-system-names'>
-<!ENTITY url.rpmfind "http://www.rpmfind.net/">
+<!ENTITY url.iana.system-names 'https://www.iana.org/assignments/operating-system-names'>
+<!ENTITY url.rpmfind "https://www.rpmfind.net/">
 <!ENTITY url.rpmfind.pbone "http://rpm.pbone.net/">
-<!ENTITY url.rrd "http://www.mrtg.org/rrdtool/">
+<!ENTITY url.rrd "https://www.mrtg.org/rrdtool/">
 <!ENTITY url.sambar "http://www.sambar.com/">
 <!ENTITY url.seaslog.document "https://seasx.github.io/SeasLog/">
-<!ENTITY url.sed "http://www.gnu.org/software/sed/sed.html">
-<!ENTITY url.setlocale.lang.win32 "http://msdn.microsoft.com/en-us/library/39cwe7zf&#37;28v=vs.90&#37;29.aspx">
-<!ENTITY url.setlocale.country.win32 "http://msdn.microsoft.com/en-us/library/cdax410z&#37;28v=vs.90&#37;29.aspx">
+<!ENTITY url.sed "https://www.gnu.org/software/sed/sed.html">
+<!ENTITY url.setlocale.lang.win32 "https://msdn.microsoft.com/en-us/library/39cwe7zfhttp://msdn.microsoft.com/en-us/library/39cwe7zf&#37#37;28v=vs.90&#37;29.aspx">
+<!ENTITY url.setlocale.country.win32 "https://msdn.microsoft.com/en-us/library/cdax410zhttp://msdn.microsoft.com/en-us/library/cdax410z&#37#37;28v=vs.90&#37;29.aspx">
```

Comment:
These URLs needs to be fixed. They currently contain both the HTTP and HTTPS versions.

```suggestion
<!ENTITY url.setlocale.lang.win32 "https://msdn.microsoft.com/en-us/library/39cwe7zf">
<!ENTITY url.setlocale.country.win32 "https://msdn.microsoft.com/en-us/library/cdax410z">
```

Reply:
> These URLs needs to be fixed. They currently contain both the HTTP and HTTPS versions.
> 
> ```suggestion
> <!ENTITY url.setlocale.lang.win32 "https://msdn.microsoft.com/en-us/library/39cwe7zf">
> <!ENTITY url.setlocale.country.win32 "https://msdn.microsoft.com/en-us/library/cdax410z">
> ```